screens/LoginScreen.tsx

```diff
@@ -38,27 +36,21 @@ export default function LoginScreen({ navigation }) {
   useEffect(() => {
     if (response?.type === "success") {
       const { id_token } = response.params;
-
-      const auth = getAuth();
-      const provider = new GoogleAuthProvider();
       const credential = GoogleAuthProvider.credential(id_token);
-      signInWithCredential(auth, credential);
-      setToken(id_token);
+      signInWithCredential(authContext.auth, credential);
     }
   }, [response]);
-  
+
   async function getToken() {
-    const resp = await postRequest(loginPath, { username, password });
-    setToken(resp.token);
-    setItem("userToken", resp.token);
+    signInWithEmailAndPassword(authContext.auth, email, password);
```

Comment:
Something to maybe think about doing later/whenever we have a sec is:

```
setLoading(true);
await signInWithEmailAndPassword(authContext.auth, email, password);
setLoading(false);
```

Where if we have loading set we show a `<Spinner/>` for the user. But this request should be pretty quick so no need to worry about it too much yet